tests/ha/remove_node.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Remove a node both by its hostname and ip address
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+
+sub run {
+    my $cluster_name = get_cluster_name;
+    my $node_01      = choose_node(1);
+    my $node_02      = choose_node(2);
+    my $node_01_ip   = get_ip($node_01);
+    my $node_02_ip   = get_ip($node_02);
+    my $remove_cmd   = 'crm cluster remove -y -c';
+    my $join_cmd     = 'crm cluster join -y -w /dev/watchdog -i eth0 -c';
+    my $timer        = '5';
+
+    # Waiting for the other nodes to be ready
+    barrier_wait("REMOVE_NODE_BY_HOST_INIT_$cluster_name");
+
+    # Remove the second node by its hostname
+    assert_script_run "$remove_cmd $node_02" if is_node(1);
+    # Need to wait a bit for cluster configuration refresh
+    sleep $timer;
```

Comment:
@alvarocarvajald @ldevulder, I'm not satisfied about the "sleep" command usage. I'd like to know if you have a better idea. 
I didn't have any issue with a 2 nodes cluster but a timer must be used with a 3 nodes cluster (or more) to let the cluster configuration refresh.

Reply:
I can't think of a better option than the `sleep`. You could try with a loop that checks for node 2 in the output of `crm configure show`, but this will not work easily in node 2 since it's the one being removed and `crm configure show` will eventually fail, so you will need to have extra code to handle such case. I think in this case the `sleep` is easier to read and to maintain, and if we run into performance issues, we can increase the time with `TIMEOUT_SCALE`.